Mathlib/Order/Lattice.lean

```diff
@@ -1169,10 +1169,12 @@ theorem map_sup [SemilatticeSup β] (hf : MonotoneOn f s) (hx : x ∈ s) (hy : y
     first
     | assumption
     | simp only [*, sup_of_le_left, sup_of_le_right]
+#align monotone_on.map_sup MonotoneOn.map_sup
```

Comment:
bors r-

I think this belongs in #6339, which is a forward-port of these lemmas.

Reply:
OK, I have removed these aligns. (Since I was already fixing some aligns, when I saw [this question](https://leanprover.zulipchat.com/#narrow/stream/270676-lean4/topic/mathport.20error) it made sense to me to also add these).